cafebabel/static/js/tag-detail.js

```diff
@@ -2,34 +2,22 @@ const container = document.querySelector('.tag-header')
 const imgUrl = container.getAttribute('data-image')
 
 addBackgroundImage(container, imgUrl)
-parallax(imgUrl)
+parallax()
 
-function parallax(imgUrl) {
+function parallax() {
   const container = document.querySelector('.tag-header.image')
-  const height = getHeight(imgUrl)
-
   if (!container) return
   window.addEventListener('scroll', () => {
     const scrolledHeight = window.pageYOffset
-    const top = container.offsetTop - height * 2
-    container.style.backgroundPositionY = `${(top - scrolledHeight) / 3}px`
+    container.style.backgroundPositionY = 50 + 0.2 * scrolledHeight + '%'
   })
 }
 
 function addBackgroundImage(container, imgUrl) {
-  if (imgUrl) {
+  if (imgUrl === 'None') {
```

Comment:
You should not have to compare to `None` which is a python keyword, if you have that value within the template please submit a bug or turn it explicitly to an empty string to check the existence here instead of comparing to that string.

Reply:
That's exactly why I compared `imgUrl` with `None`. Cause I couldn't get it with `if !value` or something like that. How can I force `value=""`?

Reply:
In the template, adding `{{ article.property or '' }}`

Reply:
done.